packages/react/src/context/use-context-mutator.ts

```diff
@@ -0,0 +1,46 @@
+import { useCallback, useContext, useRef } from 'react';
+import type { EvaluationContext } from '@openfeature/web-sdk';
+import { OpenFeature } from '@openfeature/web-sdk';
+import { Context } from '../common';
+
+export type ContextMutationOptions = {
+  /**
+   * Apply changes to the default context instead of the domain scoped context applied at the <OpenFeatureProvider/>.
+   * Note, if the <OpenFeatureProvider/> has no domain specified, the default is used.
+   */
+  default?: boolean;
+};
+
+export type ContextMutation = {
+  /**
+   * A function to set the desired context (see: {@link ContextMutationOptions} for details).
+   * @param updatedContext
+   * @returns
+   */
+  setContext: (updatedContext: EvaluationContext) => Promise<void>
+};
+
+/**
+ * Get function(s) for mutating the evaluation context associated with this domain, or the default context if `global: true`.
```

Comment:
this `global: true` bit will need to line up with your finalize name in `ContextMutationOptions`

Reply:
Nice catch!

Reply:
fixed